.github/workflows/tf-apply.yaml

```diff
@@ -98,3 +105,16 @@ jobs:
           var_file: ${{ env.TF_VAR_FILES }}
           variables: ${{ env.TF_VARS }}
           auto_approve: true
+
+
+      - name: Create Summary Output
+        id: tf_outputs_summary
+        run: |
```

Comment:
@Davidoutz [minor] Can we run this conditionally based on the presence of outputs? 

I can imagine there are repos where we don't have outputs; it'd be nice to not print anything in that case.

Something like
```yaml
if: steps.tf_apply.outputs != {}
```